scene/resources/visual_shader.cpp

```diff
@@ -4323,6 +4323,23 @@ void VisualShaderNodeFrame::_bind_methods() {
 VisualShaderNodeFrame::VisualShaderNodeFrame() {
 }
 
+////////////// Comment (Deprecated)
+
+void VisualShaderNodeComment::_bind_methods() {
```

Comment:
```suggestion
#ifndef DISABLE_DEPRECATED
void VisualShaderNodeComment::_bind_methods() {
```
And after

Reply:
Hhm, I thought I did this, looks like I'm a bit puzzled today :)